docs/reference/ccr/upgrading.asciidoc

```diff
@@ -0,0 +1,48 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-upgrading]]
+== Upgrading clusters
+
+In case of upgrading clusters, clusters that are actively using CCR, require a
```

Comment:
Sorry, I should have noticed this before. Let's use `{ccr}`. Let's reword this first sentence to: `Clusters that are actively using {ccr} require a careful approach to upgrades.`

Reply:
pushed: e7c6f01510b03010f7082f1b9e94082b0baa1f71